socialhome/tests/test_viewsets.py

```diff
@@ -25,3 +25,41 @@ def test_image_upload(self):
         self.assertIn("http://127.0.0.1:8000/media/uploads/", url)
         image_upload = ImageUpload.objects.filter(user=self.user).first()
         self.assertEqual(image_upload.image.name, url.replace("http://127.0.0.1:8000/media/", ""))
+
+
+class TestObtainSocialhomeAuthToken(SocialhomeAPITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.user = UserFactory()
+
+    def test_user_missing(self):
+        response = self.client.post(reverse("api-token-auth"))
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.data["username"], ["This field is required."])
+
+    def test_password_missing(self):
+        response = self.client.post(reverse("api-token-auth"))
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.data["password"], ["This field is required."])
+
+    def test_unable_to_login(self):
+        response = self.client.post(reverse("api-token-auth"),
+                                    {"username": self.user.username, "password": "%sLol" % self.user.password})
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.data, {'non_field_errors': ['Unable to log in with provided credentials.']})
+
+    def test_authentication(self):
+        self.client.force_authenticate(self.user, token="0ab231f9f7ab63a6f78fd9746b9bdc9926a734fd")
```

Comment:
You're testing login so you shouldn't be forcing authentication in the test - this line shouldn't be here imho.